README.md

```diff
@@ -37,80 +40,84 @@ We have done this via a bot GitHub user account (e.g. dandibot).
-   Specifically, this involves:
-   - `group_vars/all`
-   - `config.yaml.j2`
+    Required vault values:
+       - github_client_id (From GH OAuth app)
+       - github_client_secret (From GH OAuth app)
+       - aws_certificate_arn (From aws certificate manager)
+       - dummypass (this is a string password you can use for testing without GitHub authentication)
+          TODO(satra, how do we use this password later?)
+       - danditoken TODO(satra, I just made up a string, how is this used?
+           PS it is necessary to set because this var is in config.yaml.j2)
 
-   For the latter, this may involve adjusting authentication steps and profiles.
+    1. Also note that `namespace` has to be unique across any JH
+       instances created with this setup.
 
-   Also note that the namespace has to be unique across any JH
-   instances created with this setup. 
+1. Ensure `dandi-info/z2jh.yaml` uses the `ig-policy` in the file. (This
+   is not necessary to change if there is already an instance of the
+   policy in AWS. If you need to create `ig-policy` use the following:
 
-6. Create policy `ig-policy` and copy this ARN below:
-```
-{
-    "Version": "2012-10-17",
-    "Statement": [
+        ```
         {
-            "Effect": "Allow",
-            "Action": [
-                "autoscaling:DescribeAutoScalingGroups",
-                "autoscaling:DescribeAutoScalingInstances",
-                "autoscaling:DescribeLaunchConfigurations",
-                "autoscaling:DescribeTags",
-                "autoscaling:SetDesiredCapacity",
-                "autoscaling:TerminateInstanceInAutoScalingGroup",
-                "ec2:DescribeLaunchTemplateVersions",
-                "ec2:DescribeInstanceTypes"
-            ],
-            "Resource": "*"
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "autoscaling:DescribeAutoScalingGroups",
+                        "autoscaling:DescribeAutoScalingInstances",
+                        "autoscaling:DescribeLaunchConfigurations",
+                        "autoscaling:DescribeTags",
+                        "autoscaling:SetDesiredCapacity",
+                        "autoscaling:TerminateInstanceInAutoScalingGroup",
+                        "ec2:DescribeLaunchTemplateVersions",
+                        "ec2:DescribeInstanceTypes"
+                    ],
+                    "Resource": "*"
+                }
+            ]
         }
-    ]
-}
-```
-7. Update `dandi-info/z2jh.yaml` to reflect the new policy ARN. Search for `ig-policy` in the file.
+        ```
 
-To use this z2jh-aws-ansible repo for Dandihub deployment, make sure it is populated (otherwise, 
-do `git submodule update --init z2jh-aws-ansible` in the main repo before the `cd` step below):
+1. Populate (or update) the `z2jh-aws-ansible` submodule (from repo root) `git submodule update --init z2jh-aws-ansible`
```

Comment:
this and the following section should be removed, since we no longer have submodules.